plugins/scaffolder-actions/scaffolder-backend-module-http-request/README.md

```diff
@@ -80,14 +84,52 @@ spec:
         headers:
           test: 'hello'
           foo: 'bar'
-        body:
-          name: 'test'
-          bar: 'foo'
 
   output:
     getResponse: '{{ steps.backstage_request.output.body }}'
     getCode: '{{ steps.backstage_request.output.code }}'
     getHeaders: '{{ steps.backstage_request.output.headers }}'
 ```
 
+### Example of using POST method
+
+If method requires to have request body if can be specified as `body` parameters. There are two options here:
```

Comment:
Can I suggest changing this sentence to this: "If the request requires a body, it can be specified using the `body` parameter. There are two options here:"

Reply:
Changed, thanks for rephrasing it :)